feat(pfe-navigation): report pfe-nav's height in a global CSS variable (#732)
patternfly/patternfly-elements: elements/pfe-navigation/README.md

- `pfe-menu-label`: Translation for the Menu label on the main navigation dropdown.
- `pfe-full-width`: Allows the navigation to span the full width of the viewport.

### CSS Variables

>  - `--pfe-navigation__Height--actual`: when pfe-navigation initializes, it will create a global (on `body`) CSS variable that contains the height of the pfe-navigation item.  The primary uses for this is calculating the offset for anchor links, and for positioning a sticky sub-header below the pfe-navigation.  Note that multiple pfe-navigation elements will write the same variable, unless...

Reviewer: +1 Chris this should be dash dash Height

Reviewer 2: Cool. Fixed.

>  - `--pfe-navigation--Height--actual`: when pfe-navigation initializes, it will create a global (on `body`) CSS variable that contains the height of the pfe-navigation item.  The primary uses for this is calculating the offset for anchor links, and for positioning a sticky sub-header below the pfe-navigation.  Note that multiple pfe-navigation elements will write the same variable, unless...

Reviewer 3: @chrisdo1 @castastrophe @starryeyez024 - Is someone going to finish this sentence?

![b0f76f61e0b8ee52a5e77da9371b5c93--the-lorax-sweet-tattoos](https://user-images.githubusercontent.com/330256/74740363-b65dee00-5228-11ea-9c2c-d23bcd53292f.jpg)

Reviewer: The bullet below addresses it.

Reviewer 4: LOL I thought the same thing when I read it, it took me a moment to realize the nested bullet answered the question. I think maybe a colon instead of an ellipsis would read better?

Reviewer 2: changed this to a colon.

>  - `--pfe-navigation--Height--actual`: when pfe-navigation initializes, it will create a global (on `body`) CSS variable that contains the height of the pfe-navigation item.  The primary uses for this is calculating the offset for anchor links, and for positioning a sticky sub-header below the pfe-navigation.  Note that multiple pfe-navigation elements will write the same variable, unless...

Reviewer 4: ```suggestion
 - `--pfe-navigation--Height--actual`: when pfe-navigation initializes, it will create a global (on `body`) CSS variable that contains the height of the pfe-navigation item.  The primary uses for this is calculating the offset for anchor links, and for positioning a sticky sub-header below the pfe-navigation.  Note that multiple pfe-navigation elements will write the same variable, unless:
```

Reviewer 2: fixed

>  - `--pfe-navigation__${ID}--Height--actual`: like the previous varaible, but if the pfe-navigation has an `id` attribute, it will be appended to the CSS variable name.  This makes it possible to distinguish the heights of multiple pfe-navigation elements on the same page.

Reviewer: Variable is also misspelled here

Reviewer 2: fixed

>  - `--pfe-navigation__${ID}--Height--actual`: like the previous varaible, but if the pfe-navigation has an `id` attribute, it will be appended to the CSS variable name.  This makes it possible to distinguish the heights of multiple pfe-navigation elements on the same page.

Reviewer 4: ```suggestion
 - `--pfe-navigation__${ID}--Height--actual`: like the previous variable, but if the pfe-navigation has an `id` attribute, it will be appended to the CSS variable name.  This makes it possible to distinguish the heights of multiple pfe-navigation elements on the same page.
```

Reviewer 2: fixed

- `pfe-menu-label`: Translation for the Menu label on the main navigation dropdown.
- `pfe-full-width`: Allows the navigation to span the full width of the viewport.

### CSS Variables

>  - `--pfe-navigation--ReportedHeight`: when pfe-navigation initializes, it will create a global (on `body`) CSS variable that contains the height of the pfe-navigation item.  The primary uses for this is calculating the offset for anchor links, and for positioning a sticky sub-header below the pfe-navigation.  Note that multiple pfe-navigation elements will write the same variable, unless...

Reviewer: ```suggestion
 - `--pfe-navigation--Height--actual`: when pfe-navigation initializes, it will create a global (on `body`) CSS variable that contains the height of the pfe-navigation item.  The primary uses for this is calculating the offset for anchor links, and for positioning a sticky sub-header below the pfe-navigation.  Note that multiple pfe-navigation elements will write the same variable, unless...
```

Reviewer 4: why "actual"?

Reviewer 4: I'm going to assume the answer is yes & close my own question :)

>  - `--pfe-navigation--ReportedHeight-${ID}`: like the previous varaible, but if the pfe-navigation has an `id` attribute, it will be appended to the CSS variable name.  This makes it possible to distinguish the heights of multiple pfe-navigation elements on the same page.

Author: Reminder to update the variable names in the README to `Height--actual` or whatever we settle on.

Reviewer 2: Updated to include correct variable name

>  - `--pfe-navigation--ReportedHeight-${ID}`: like the previous varaible, but if the pfe-navigation has an `id` attribute, it will be appended to the CSS variable name.  This makes it possible to distinguish the heights of multiple pfe-navigation elements on the same page.

Reviewer: ```suggestion
 - `--pfe-navigation__${ID}--Height--actual`: like the previous variable, but if the pfe-navigation has an `id` attribute, it will be appended to the CSS variable name.  This makes it possible to distinguish the heights of multiple pfe-navigation elements on the same page.
```

Reviewer 2: @castastrophe updated lines 72 and 73 to show the correct variable name.
